app/controllers/chatfuel_controller.rb

```diff
@@ -53,6 +53,26 @@ def json_response
           }
         }
       ]
+    elsif @question_answer && !@next_question_answer
+      topic = @text_component.topic
+      messages =  [
+          {
+          attachment: {
+            payload: {
+              template_type: "button",
+              text: "#{content}",
+              buttons: [
+                {
+                  type: "show_block",
+                  block_name: "cont_" + topic.name,
+                  title: "Zurück",
+                }
+            ]
+            },
+            type: "template"
+          }
+        }
+      ]
     else
```

Comment:
For your information, the next time we update, all our `belongs_to` associations will be required by default: http://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html